Update Philips 929003017102 Hue Wall Switch Module docs (#2464)
Koenkk/zigbee2mqtt.io: docs/devices/929003017102.md

### Pairing
Press the reset button for ten seconds to reset the device - the red LED flashes one time confirming the reset. Then short the pins of input one.  
The red LED begins to flash every two seconds indicating pairing mode.
Out of the box, the device enters pairing mode, as soon as the pins of input 1 are shortened - the red LED starts to flash.
If the device has not been paired yet, it can also be put into pairing mode by short pressing the reset button.

### Directly control Zigbee groups
It's possible to configure this switch to directly control Zigbee groups. In this way, the switch can control e.g. a group of light bulbs even when Zigbee2MQTT is not running, and it can also make controlling the lights feel a bit more responsive. This switch can also act as a dimmer: for each successive long press of the switch, it will alternate between increasing and decreasing brightness. To set this all up, follow these steps in the Zigbee2MQTT frontend:
1. Go to the *Groups* tab and create a new group if it does not already exist. Add devices to the groups that you want to be controlled by this switch.

> 1. Add this device to the group. Note that this device will not show up in the group, but this step is important.

Reviewer: Since you are at it, can you make this change? I believe the wall switch shows up in the group (at least does in my case)

```suggestion
1. Add this Hue wall switch to the group.
```

Author: I just checked my group which is controlled by the Hue Wall Switch Module, and I can't see the device being in the group. Maybe it only shows up just after being added? And probably disappears after a restart? I'll validate this theory soon.

Author: I tried it, and I couldn't get the device to show up in the group.

Seemingly the device doesn't respond to the group add request, but handles it silently. I got an error every time I tried. I only copied the last error log, but I think it was the same every time:

```
Failed to add from group (Command 0x001788010cc233b8/1 genGroups.add({"groupid":111,"groupname":""}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 54334 - 1 - 70 - 4 - 0 after 10000ms))
```

Author: And I don't remember it ever showing up in the group.

Reviewer: In my case it does, so I suggest to use:

```
1. Add this device to the group. Note that this device might not show up in the group, but this step is important.
```

Author: I applied your latest suggestion and the first part of your original suggestion.
